### ISSUE TITLE
Compile succeeded but the terminal window was killed immediately after running the exe, under debug mode

### ISSUE THREAD

**Issue by breadli428** [cmm-21/a0#4]
Hi,

Thanks for Dongho's help I was finally able to build and run. This works perfectly under the release mode of CMake and I can see the ball dragging demo window for the a0 assignment.

However, the demo window appeared and got killed immediately after showing up in the debug mode.

It would be super appreciated if anyone who has similar experience could provide a hint.

Thanks,
Chenhao

**Comment by moritzge** [comment]
What happens after the window disappears? Does it exit with a debug assertion?

**Comment by breadli428** [comment]
In the terminal after running Projects\a0-breadli428\build\src\app\Debug\app.exe?

No, nothing is returned.

**Comment by breadli428** [comment]
In the output, the exit code of build is 0.

**Comment by breadli428** [comment]
I can also see this

The program '[7364] app.exe' has exited with code -1073741515 (0xc0000135).

**Comment by breadli428** [comment]
Hi,

I finally solved my problem. Here is what I found and hope this could help you if you have the similar question.

I ran into the \build\src\app\Debug folder and checked if I could open the app.exe manually. Then it said ucrtbased.dll could not be found. However I can find it under C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x86\ucrt.

And I searched and found this is because a failure of Package Win10SDK_10.0.19041, which should be installed when installing Visual Studio. And I actually did.

But what I did to finally repair this is go to the control panel and repair the Visual Studio, by first unchecking and then checking the Win10SDK_10.0.19041 in the VS installer. Then finally everything works!!!

Thank you and all TAs that helped me with setting up the environment.

Chenhao
